experimental/framework/tinylicious-client/src/test/TinyliciousClient.spec.ts

```diff
@@ -5,42 +5,364 @@
 
 import { strict as assert } from "assert";
 import { v4 as uuid } from "uuid";
-import { SharedMap } from "@fluid-experimental/fluid-framework";
+import { DiceRoller } from "@fluid-example/diceroller";
+import { SharedMap, SharedDirectory } from "@fluid-experimental/fluid-framework";
 import { ContainerSchema } from "@fluid-experimental/fluid-static";
 import {
-    TinyliciousClient,
     TinyliciousConnectionConfig,
     TinyliciousContainerConfig,
 } from "..";
+import { ResettableTinyliciousClient } from "./ResettableTinyliciousClient";
 
-describe("TinyliciousClient", () => {
+describe("TinyliciousClient pre-initialization", () => {
+    let documentId: string;
+    beforeEach(() => {
+        documentId = uuid();
+    });
+
+    /**
+    * Scenario: test if TinyliciousClient can get a container without being initialized.
+    *
+    * Expected behavior: an error should be thrown when trying to get a container
+    * without initializing TinyliciousClient.
+    */
+    it("cannot get a container without initializing", async () => {
+        const containerConfig: TinyliciousContainerConfig = { id: documentId };
+        const schema: ContainerSchema = {
+            name: documentId,
+            initialObjects: {
+                map1: SharedMap,
+            },
+        };
+        const containerAndServices = ResettableTinyliciousClient.getContainer(containerConfig, schema);
+        const errorFn = (error) => {
+            assert.notStrictEqual(error.message, undefined, "TinyliciousClient error is undefined");
+            return true;
+        };
+
+        await assert.rejects(
+            containerAndServices,
+            errorFn,
+            "TinyliciousClient can get a container without initialization",
+        );
+    });
+
+    /**
+    * Scenario: test if TinyliciousClient can create a container without being initialized.
```

Comment:
Nit: Convention for whitespace formatting here is to align the asterisks, e.g.
```typescript
    /**
     * A comment
     */
```
Here and in other instances

Reply:
learning something new everyday! Updated.